### PR TITLE
[Gallery layouts] Update storybook for gallery layouts

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Introduce storybook docs for gallery layouts for components and composites
# Why

documentation for gallery layouts

https://skype.visualstudio.com/SPOOL/_workitems/edit/3417260
https://skype.visualstudio.com/SPOOL/_workitems/edit/3417262
# How Tested

inspected locally check link for 
- video gallery
- Call composite

### CODE REVIEW

File: packages/storybook/stories/CallComposite/CallCompositeDocs.tsx [hunk 1]
```diff
@@ -271,7 +271,55 @@ export const Docs: () => JSX.Element = () => {
         gallery](./?path=/docs/ui-components-videogallery--video-gallery) component docs for more information on our
         local video tile and some of the other options we have for the local video tile when just using the components.
       </Description>
-
+      <Heading>Customizing the Gallery Layout</Heading>
```
Comment (JamesBurnside) [inline packages/storybook/stories/CallComposite/CallCompositeDocs.tsx:274]: Nit: mention default
```suggestion
      <Heading>Customizing the default Gallery Layout</Heading>
```

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx [hunk 2]
```diff
@@ -93,6 +119,27 @@ const getDocs: () => JSX.Element = () => {
       <Canvas mdxSource={FloatingLocalVideoExampleText}>
         <FloatingLocalVideoExample />
       </Canvas>
+      <Subheading>Speaker layout</Subheading>
+      <SingleLineBetaBanner></SingleLineBetaBanner>
+      <Description>
+        This Layout is meant to highlight the current dominant speaker in the call. For this view in the video gallery
```
Comment (JamesBurnside) [inline packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:125]: nit: For new paragraphs/sections, avoid `this`
```suggestion
        Speaker Layout is meant to highlight the current dominant speaker in the call. For this view in the video gallery
```

**Comment by JamesBurnside** [comment]
Storybook controls don't seem to have all the options:
image

Also can we remove this layout control:
image

**Review by JamesBurnside — APPROVED** [review]
QoL updates just, LGTM

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**